samples/metrics_monitor/sample/cttmetrics_sample.cpp

```diff
@@ -45,28 +54,67 @@ void signal_handler(int signo)
     }
 }
 
+static void
+usage(const char *appname)
+{
+    printf("metrics_monitor - Monitors GPU usage per engine\n"
+        "\n"
+        "Usage: %s [OPTION]\n"
+        "\n"
+        "\tThe following parameters are optional:\n\n"
+        "\t[-h]          Show this help text.\n"
+        "\t[-s <num>]    Number of metric samples to collect during sampling period(valid range %u..%u, default %u).\n"
+        "\t[-p <ms>]     Sampling period in milliseconds(valid range %u..%u, default %u).\n"
+        "\t[-d <path>]   Path to gfx device (like /dev/dri/card* or /dev/dri/renderD*).\n"
```

Comment:
What's is program default behavior for device? I suggest to have some text describing app behavior in the very end after options.